common/src/Logging.ProvisioningTransport.Common.cs

```diff
@@ -10,20 +10,21 @@ internal sealed partial class Logging : EventSource
     {
         private const int RegisterDeviceId = 12;
         private const int OperationStatusLookupId = 13;
-        
+
         [NonEvent]
         public static void RegisterDevice(
             object thisOrContextObject,
-            string registrationId, 
+            string registrationId,
             string idScope,
-            string attestationType, 
-            string operationId, 
-            TimeSpan? retryAfter, 
+            string attestationType,
+            string operationId,
+            TimeSpan? retryAfter,
             string status)
         {
             DebugValidateArg(thisOrContextObject);
             DebugValidateArg(attestationType);
             DebugValidateArg(retryAfter);
+
```

Comment:
I wonder if the `if (IsEnabled)` check shouldn't be before the DebugValidateArgs.

Reply:
you are right, it should be